javascript/new-src/src/js/app.js

```diff
@@ -328,14 +328,15 @@ angular
     if (typeof $scope.Data.id === 'number' &&
         $scope.Data.add_ports !== '' &&
         $scope.Universes.indexOf($scope.Data.id) === -1) {
-     if ($scope.Data.name === '') {
+     if ($scope.Data.name === '' || $scope.Data.name === undefined) {
```

Comment:
These ought to be the other way round I'd say, otherwise you're doing a string comparison on a null.